HangPerson/HangPerson/main.m

```diff
@@ -11,8 +11,178 @@
+                            printf("|        /|\\  \n");
+                            printf("|        / \\  \n");
+                            printf("\n");
+                            break;
+                        case 6:
+                            printf(" _________     \n");
+                            printf("|         |    \n");
+                            printf("|         0    \n");
+                            printf("|        /|\\  \n");
+                            printf("|        / \\  \n");
+                            printf("|              \n");
+                            printf("\n");
+                            break;
+                        case 7:
+                            printf(" _________     \n");
+                            printf("|         |    \n");
+                            printf("|         0    \n");
+                            printf("|        /|\\  \n");
+                            printf("|        / \\  \n");
+                            printf("|              \n");
+                            printf("|              \n");
+                            printf("\n");
+                            break;
+                        default:
+                            printf("...");
+                    }
+                }
+                
+                
+                // ** checks for the correct match
+                for (int i = 0; i < count; i++){
+                    if (c == levelOne[i]){
+                        //printf("There is a match!\n");
+                        //printf("Guess: %c matches %c \n", c, levelOne[i]);
+                        match++;
+                        //printf("match number: %d", match);
+                        levelOneMatch[i] = c;
+                        tries--;
+                    }
+                }
+                
+                
+                // ** prints out the correct matches
+                for (int i = 0; i < count; i++){
+                    printf("%c ", levelOneMatch[i]);
+                }
+                
+                
+                // ** checks if player guessed the same char twice
+                levelOneGuesses[tries] = c;
+                s = strstr(levelOneGuesses, &c);
+                if (s != NULL){
+                    printf("You already guessed this letter!\nPlease pick another letter.\n");
+                }
+                
+                
+                // ** increases the count for tries
+                tries++;
+                printf("\n\nYou have %d tries left\n", maxTries - tries);
+                
+                
+                
+                // ** checks to see if the user's array matches the OG word's array
+                if ( levelOneMatch[0] == levelOne[0] && levelOneMatch[1] == levelOne[1] && levelOneMatch[2] == levelOne[2] && levelOneMatch[3] == levelOne[3] && levelOneMatch[4] == levelOne[4] && levelOneMatch[5] == levelOne[5] ){
```

Comment:
The one thing I would suggest is that you make this more general. Use a for loop and compare within the loop...
```c
int won = 1;
for (int i = 0; i < strlen(levelOne); i++) {
    if (levelOne[i] != levelOneMath[i]) {
        won = 0;
        break;
    } 
}

if (won) {
    printf("YOU ARE THE BEST EVER!!");
}
```